gradle/verification-metadata.xml

```diff
@@ -2506,121 +2506,241 @@
             <sha256 value="015d5c229f3cd5c0ebf175c1da08d596d94043362ae9d92637d88848c90537c8" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.apache.lucene" name="lucene-analysis-common" version="9.6.0">
+         <artifact name="lucene-analysis-common-9.6.0.jar">
+            <sha256 value="f2f0d724c99440f4ab9304c06e7966653d088e872a2bbd709605dff547e5a494" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="org.apache.lucene" name="lucene-analysis-common" version="9.6.0-snapshot-246ac4bcbe6">
          <artifact name="lucene-analysis-common-9.6.0-snapshot-246ac4bcbe6.jar">
             <sha256 value="1a361c5114b80c60174577af2b4f0d568c0b1de8ffb09a3512b27e0bd209362a" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.apache.lucene" name="lucene-analysis-icu" version="9.6.0">
```

Comment:
Could you remove the old entries too? We should try to house keep old enrtries when dding new ones or the file becomes huge.

Reply:
that's a good point, I guess we should add this to the upgrade instructions. I've seen this done inconsistently in the github history with previous upgrades.

Reply:
It is actually here: https://github.com/elastic/elasticsearch/blob/main/BUILDING.md#make-a-change-in-the-build

```
In case of updating a dependency, ensure to remove the unused entry of the outdated dependency manually from the verification-metadata.xml file.
```
Very difficult to find though.